pkg/action/executor/k8s_controller.go

```diff
@@ -176,3 +169,67 @@ func (c *parentController) shouldSkipOperator(controller *unstructured.Unstructu
 func (rc *parentController) String() string {
 	return rc.name
 }
+
+//To determine if ORM should be used or not in terms of the owner reference of the work load controller
+func (c *parentController) shouldUseOrm(controller *unstructured.Unstructured) bool {
```

Comment:
The naming usage here is mixed and confusing. Lets use `obj` as the argument name. Further, you don't really need to pass it as an argument as its already available in `c *parentController`.
I understand that some of this might be technical debt, recommend correcting it while we are at it.

Reply:
Make sense, I would change it to `shouldUseOrm(obj *unstructured.Unstructured)`